libcontainer/apparmor/apparmor.go

```diff
@@ -6,17 +6,28 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 
 	"github.com/opencontainers/runc/libcontainer/utils"
 )
 
 // IsEnabled returns true if apparmor is enabled for the host.
 func IsEnabled() bool {
 	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil && os.Getenv("container") == "" {
-		if _, err = os.Stat("/sbin/apparmor_parser"); err == nil {
+		isEnabledByParameters := func() bool {
 			buf, err := ioutil.ReadFile("/sys/module/apparmor/parameters/enabled")
 			return err == nil && len(buf) > 1 && buf[0] == 'Y'
```

Comment:
I think `bytes.HasPrefix(buf, []byte{'Y'})` is a bit less awful to read.

Reply:
Yep, changed the check to check for the "Y" prefix to not change the behavior.

Reply:
Me, too, thought about `HasPrefix`, and decided that `len(buf) > 1 && buf[0] == 'Y'` is good enough :)
Anyway, HasPrefix is good enough, too.